crates/agent/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:noble
```

Comment:
Does the image here affect our costs in any noticable way? I'm wondering if we could save costs by using a slimmer image. I guess it depends on how frequently this image is going to be run by Cloud Run and how much time is taken by the image to boot up compared to other images 🤔

Reply:
Not especially, but this isn't a huge image (50MB iirc?). I do think it's a good idea to use the same OS base as our GitHub runner environment.